test/boulder-config.json

```diff
@@ -55,7 +55,7 @@
   "ca": {
     "serialPrefix": 255,
     "profile": "ee",
-    "debugAddr": "localhost:8001",
+    "debugAddr": "0.0.0.0:8001",
```

Comment:
I worry about exposing debug port on all interfaces (when run on the host instead of container), but I'm not sure how actually dangerous it is on a dev box. If someone was able to hit the debug port on my dev instance, what would they be able to accomplish? @jmhodges?